### PR TITLE
Added a navbar component with a Home link.

### PR DESCRIPTION
**Pull request by thehappyman**
- Added the Navbar component with Home link
- Added the Navbar component to the App.js

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
+import { BrowserRouter } from "react-router-dom";
+import { Navbar } from "./components/Navbar";
+
 function App() {
-  return <h1 className="text-3xl font-bold underline">Hello world!</h1>;
+  return (
+    <BrowserRouter>
+      <Navbar />
+      <h1 className="text-3xl font-bold underline">Hello world!</h1>;
```
Comment (gitsoufiane) [inline src/App.js]: remove this line

File: package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "react-dom": "^18.1.0",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "router-dom": "^2.2.9",
```
Comment (gitsoufiane) [inline package.json]: remove this line , we don't  use this package

**Review by gitsoufiane — CHANGES_REQUESTED** [review]
please check my comments

**Comment by thehappyman** [comment]
Changes are done.
please check them.

**Review by gitsoufiane — APPROVED** [review]
👍

**Pull request merged**